Cargo.toml

```diff
@@ -579,7 +579,7 @@ tree-sitter = { version = "0.25.6", features = ["wasm"] }
 tree-sitter-bash = "0.25.0"
 tree-sitter-c = "0.23"
 tree-sitter-cpp = "0.23"
-tree-sitter-css = "0.23"
+tree-sitter-css = { git = "https://github.com/mantou132/tree-sitter-css", rev = "62bb39a376ee8aa0ceb6cab27474b7c7136913b9" }
```

Comment:
Little question, don't you intend to contribute this change to the upstream? 🤔
https://github.com/tree-sitter/tree-sitter-css/compare/master...mantou132:tree-sitter-css:master

Reply:
This changes:

- Supports `\`` at the beginning and end of the document
- Recognize `${}` content as comments and avoid recognizing it as a pseudo-class

These are not part of the CSS spec, so I don't think they should be merged upstream.

Reply:
Is there any better solution besides modifying tree-sitter-css? Although I've tried creating a PR for tree-sitter-css

Reply:
Sorry, I have thought of it more and more and could not find anything clever than "move it to zed-industries org".

I do no think it's a good solution either, maybe @notpeter have ideas?